### PR TITLE
Issue #3213908: Document process to PR changes to sector-project

### PR DESCRIPTION
**Pull request by xurizaemon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,27 @@ For a better idea as to what Sector does, what features it has, and why you shou
...
+      "source": {
+        "url": "/{path-to}/sector-project/.git",
+        "type": "git",
+        "reference": "{your-branch}"
+      }
+    }
+  }
+  ```
+* Use `composer create-project sparksinteractive/sector-project --repository-url <path-to>/repo.json <destination>` to build your project.
+* When ready, use the Drupal.org issue queue to collaborate on the proposal with a pull request to [sparksi/sector-project](https://github.com/sparksi/sector-project) on Github for the changes themselves.
```
Comment (RoSk0) [inline README.md]: Do we want to link to the issue queue itself or at least to appropriate project here?

File: README.md [hunk 2]
```diff
@@ -28,3 +28,27 @@ For a better idea as to what Sector does, what features it has, and why you shou
...
+      "version": "{version}",
+      "source": {
+        "url": "/{path-to}/sector-project/.git",
+        "type": "git",
+        "reference": "{your-branch}"
+      }
+    }
+  }
+  ```
+* Use `composer create-project sparksinteractive/sector-project --repository-url <path-to>/repo.json <destination>` to build your project.
```
Comment (RoSk0) [inline README.md]: `--repository-url` is deprecated. 

Please replace with `--repository` example:
```shell
composer create-project --repository '{"type":"path", "url":"</local/path>","options":{"symlink":false}}' --stability dev --prefer-dist sparksinteractive/sector-project <destination>
```

**Review by RoSk0 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xurizaemon) [reply README.md]: The README in this project says that PRs are accepted here. If that's not correct, we should change that text too.

https://git.drupalcode.org/project/sector / https://www.drupal.org/project/sector is the profile project.

(I'm confused here too, which is why I'm trying to establish the correct procedure and document it. If I've got it wrong, corrections welcome!)

**Comment by DanielVeza** [comment]
Looking good! 

Small thing - Should we add a bit of extra clarification on what the {} variables should actually be?

**Comment by xurizaemon** [comment]
EDIT: Yes, we should. And the rest of this comment is about **how to test changes to `sector-distribution` repo - it's fine to ignore it**. 😀 

I moved these notes to [d.o#3226591: Document process to test a change to sector-distribution](https://www.drupal.org/project/sector/issues/3226591)

**Comment by xurizaemon** [comment]
OK, it was late and I got mixed up. The comment above is "how do I test changes to `sector-distribution`, and the PR in this thread is "how do I test changes to `sector-project`". Please test the latter 👍🏼 

Proposed help text visible here: https://github.com/xurizaemon/sector-project/tree/document_forkflow

**Pull request merged**